CRITICAL_NOIR_LIBRARIES

```diff
@@ -0,0 +1,13 @@
+https://github.com/noir-lang/ec
+https://github.com/noir-lang/eddsa
+https://github.com/noir-lang/mimc
+https://github.com/noir-lang/schnorr
+https://github.com/noir-lang/noir_sort
+https://github.com/noir-lang/noir-edwards
+https://github.com/noir-lang/noir-bignum
+https://github.com/noir-lang/noir_bigcurve
+https://github.com/noir-lang/noir_base64
+https://github.com/noir-lang/noir_string_search
+https://github.com/noir-lang/sparse_array
+https://github.com/noir-lang/noir_rsa
+https://github.com/noir-lang/noir_json_parser
```

Comment:
Instead of having this list, couldn't we read the external repositories [from the Github Workflow](https://github.com/noir-lang/noir/blob/f769653b61ab6c0b669a6781a4ed81d431b309c0/.github/workflows/test-js-packages.yml#L522-L533)? [yq](https://github.com/mikefarah/yq) can be used to query the data.

Reply:
I'd prefer this as a pure data file just for ease of integration elsewhere, maybe we can have the workflow pull from this instead.

Reply:
That could work too, there is an [example](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#fromjson) in the docs. You need a step to create a JSON output for the matrix with `{"include": <critical noir libraries>}` and then use `fromJson` where the `matrix` currently is.

Reply:
I've updated the workflow to pull from the list of critical libraries.